### DISCUSSION TITLE
Atomstack A5 PRO - limit switch

### DISCUSSION THREAD

**Discussion by Hrvoje-1** [arkypita/LaserGRBL#1877] [Q&A]
Hi! There are no limit switches on Atomstack A5 PRO. How can I add them?

**Comment by Tribo-2dot0** [comment]
See where, they should be fixed and placed on their tracks at the beginning and end of the tracks.

**Reply by Hrvoje-1** [reply]
Hi! It is not a problem to find the place where the stoppers should be. I see as the problem how to wire the stopper and where the wires should go. Also, how to handle the signal from the stopper?
Thank you for your answer! :)

**Reply by Tribo-2dot0** [reply]
If this is your card, probably here:
![4caca524-cce3-46ea-aa08-d2daf0b5a3ff](https://user-images.githubusercontent.com/83982525/183236327-296f0076-82df-4632-bfc6-6ba9e3d3b3f1.jpg)

**Reply by Hrvoje-1** [reply]
I should open the casing and see what I have inside. :)
